docs/reference/ml/anomaly-detection/ml-configuring-transform.asciidoc

```diff
@@ -1,14 +1,14 @@
 [role="xpack"]
 [[ml-configuring-transform]]
-= Transforming data with script fields
+= Altering data with runtime mappings
 
-If you use {dfeeds}, you can add scripts to transform your data before
-it is analyzed. {dfeeds-cap} contain an optional `script_fields` property, where
-you can specify scripts that evaluate custom expressions and return script
-fields.
+If you use {dfeeds}, you can use runtime mappings to alter your data before it 
+is analyzed. You can add an optional `runtime_mappings` property to your 
+{dfeeds}, where you can specify field types and scripts that evaluate custom 
+expressions.
```

Comment:
I think it would be helpful to state why they'd want to do this. For example:
```suggestion
expressions without affecting the indices that you're retrieving the data from.
```